### ISSUE TITLE
4.5 개/고양이 분류에서 from_logits 관련 Warning

### ISSUE THREAD

**Issue by Haebuk** [lovedlim/tensorflow#2]
안녕하세요.
5-2-3 모델 훈련 부분(p.207-8)에서 `from_logits=True`의 의미를 마지막 출력층에서 확률 값이 아닌 레이블값을 반환할 때 쓰는 파라미터라고 이해했는데요, 실제로 돌리면 아래와 같은 경고가 나타납니다:
```
/usr/local/lib/python3.7/dist-packages/tensorflow/python/keras/backend.py:5017:
UserWarning: "`binary_crossentropy` received `from_logits=True`,
but the `output` argument was produced by a sigmoid or softmax activation and thus does not represent logits. Was this intended?"

 '"`binary_crossentropy` received `from_logits=True`, but the `output`'
```
`from_logits=True`를 지정하신 이유와, 해당 경고문이 뜨는 것에 대해 조언해주시면 감사하겠습니다:)

**Comment by tsdata** [comment]
안녕하세요. 저자 오승환입니다. 

말씀해 주신 것처럼, BinaryCrossentropy 함수에 from_logits=True를 설정하면 레이블 값을 반환하는 것이 맞습니다.  따라서, 마지막 출력층에 sigmoid 활성화함수를 지정하지 않았어야 하는데, 오류가 있는 점을 양해 부탁 드립니다. 

케라스 함수 정의를 보면, from_logits=False가 기본 설정되어 있어서, 일반적으로 마지막 출력층에 sigmoid 활성화함수를 지정하게 됩니다. 따라서, 확률값을 sigmoid 함수를 통과시키면서 레이블을 최종 출력하게 됩니다. 

요약하자면, 마지막 출력층에 sigmoid나 softmax를 설정하는 경우에는 from_logits=False로 지정하는 것이 맞습니다. 

다시 한번 불편을 드린 점에 대해서 사과 드립니다. 감사합니다.

**Comment by Haebuk** [comment]
감사합니다.
